src/Workspaces/CSharp/Portable/Utilities/CSharpTypeStyleHelper.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.CSharp.Utilities
+{
+    internal abstract partial class CSharpTypeStyleHelper
```

Comment:
This was moved down from CSharpFeatures.  Other than renaming it, it was not changed.